tech/languages/ocaml/ocaml-installation.md

```diff
@@ -0,0 +1,50 @@
+---
+title: OCaml
+subsection: ocaml
+section: tech-languages
+order: 1
+version: 1.0
+description: An open source multi-paradigm programming language and toolchain. 
+---
+
+# What is OCaml?
+
+Quoting the [ocaml.org](https://ocaml.org/) website:
+
+> OCaml is an industrial-strength programming language supporting functional, imperative and object-oriented styles.
+
+## OCaml installation
+
+To install the OCaml tools, type on a terminal:
+
+```console
+$ sudo dnf install ocaml
+```
+
+The `ocaml` and `ocamlfind`, `ocamlc` and `ocamlopt` binaries will become available on the system.
+
+# OPAM 
+
+OPAM (**O**Caml **Pa**ckage **M**anager)  is a source-based package manager for OCaml. It supports multiple simultaneous compiler installations, flexible package constraints, and a Git-friendly development workflow. 
```

Comment:
```suggestion
OPAM (**O**Caml **P**ackage **M**anager)  is a source-based package manager for OCaml. It supports multiple simultaneous compiler installations, flexible package constraints, and a Git-friendly development workflow. 
```

Reply:
The a letter makes part of the acronym.

Reply:
Sorry, I've somehow missed that! Maybe make it a capital `A` then?

Anyway, it's up to you.

Reply:
Minor nit, but let's not have trailing whitespace :)
```suggestion
OPAM (**O**Caml **Pa**ckage **M**anager)  is a source-based package manager for OCaml. It supports multiple simultaneous compiler installations, flexible package constraints, and a Git-friendly development workflow.
```